src/Components/main.css

```diff
@@ -61,7 +52,7 @@
 
 .paragraph{
     font-weight: bold;
-    font-family: Avenir;
+    font-family: 'Roboto Mono', Avenir;
```

Comment:
Just a note, Avenir is not on everyone's computer. The addition of Roboto Mono here worked wonders for the design - surprised I didn't notice earlier. It might be worthwhile to simply remove it in favor of a Roboto Mono & Poppins font stack. I'll leave this decision to you all though when we get past the content phase of things.